ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -377,6 +395,18 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         }
 
         if indexPath.row == 2 {
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: BooleanCell.identifier, for: indexPath) as? BooleanCell else {
```

Comment:
Have you tried running it on iOS 13?
On my simulators this seems to be hanging the app.

Looks like a cyclical chain of layout events:

#0	0x000000010631f8fb in BarButtonItemButton.updateLayout() at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/BarButtonItemButton.swift:86
#1	0x000000010631eaea in BarButtonItemButton.layoutSubviews() at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/BarButtonItemButton.swift:41
#2	0x000000010631eb2b in @objc BarButtonItemButton.layoutSubviews() ()
#3	0x00007fff4987a5f0 in -[UIView(CALayerDelegate) layoutSublayersOfLayer:] ()
#4	0x00007fff2b4f9260 in -[CALayer layoutSublayers] ()
#5	0x00007fff2b4ff3eb in CA::Layer::layout_if_needed(CA::Transaction*) ()
#6	0x00007fff49865670 in -[UIView(Hierarchy) layoutBelowIfNeeded] ()
#7	0x000000010654f312 in NavigationBar.update(with:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationBar.swift:544
#8	0x0000000106309c31 in NavigationController.updateNavigationBar(for:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationController.swift:142
#9	0x000000010630af1c in NavigationController.navigationController(_:willShow:animated:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationController.swift:216
#10	0x000000010630b23d in @objc NavigationController.navigationController(_:willShow:animated:) ()
#11	0x00007fff48c2131f in -[UINavigationController viewWillAppear:] ()
#12	0x00007fff48ccd71e in -[UIViewController _setViewAppearState:isAnimating:] ()
#13	0x00007fff48ccde01 in -[UIViewController __viewWillAppear:] ()
#14	0x00007fff48bd8525 in __56-[UIPresentationController runTransitionForCurrentState]_block_invoke.465 ()
#15	0x00007fff4937bb72 in _runAfterCACommitDeferredBlocks ()
#16	0x00007fff4936c300 in _cleanUpAfterCAFlushAndRunDeferredBlocks ()
#17	0x00007fff4939cb09 in _afterCACommitHandler ()
#18	0x00007fff23da1087 in __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ ()
#19	0x00007fff23d9bb3e in __CFRunLoopDoObservers ()
#20	0x00007fff23d9c08a in __CFRunLoopRun ()
#21	0x00007fff23d9b8a4 in CFRunLoopRunSpecific ()
#22	0x00007fff38c05bbe in GSEventRunModal ()
#23	0x00007fff49372964 in UIApplicationMain ()
#24	0x00000001058bcc6b in main at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/AppDelegate.swift:21
#25	0x00007fff5211c1fd in start ()
#26	0x00007fff5211c1fd in start ()

========================================================

#0	0x000000010631eae5 in BarButtonItemButton.layoutSubviews() at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/BarButtonItemButton.swift:41
#1	0x000000010631eb2b in @objc BarButtonItemButton.layoutSubviews() ()
#2	0x00007fff4987a5f0 in -[UIView(CALayerDelegate) layoutSublayersOfLayer:] ()
#3	0x00007fff2b4f9260 in -[CALayer layoutSublayers] ()
#4	0x00007fff2b4ff3eb in CA::Layer::layout_if_needed(CA::Transaction*) ()
#5	0x00007fff49865670 in -[UIView(Hierarchy) layoutBelowIfNeeded] ()
#6	0x000000010654f312 in NavigationBar.update(with:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationBar.swift:544
#7	0x0000000106309c31 in NavigationController.updateNavigationBar(for:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationController.swift:142
#8	0x000000010630af1c in NavigationController.navigationController(_:willShow:animated:) at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI/Navigation/NavigationController.swift:216
#9	0x000000010630b23d in @objc NavigationController.navigationController(_:willShow:animated:) ()
#10	0x00007fff48c2131f in -[UINavigationController viewWillAppear:] ()
#11	0x00007fff48ccd71e in -[UIViewController _setViewAppearState:isAnimating:] ()
#12	0x00007fff48ccde01 in -[UIViewController __viewWillAppear:] ()
#13	0x00007fff48bd8525 in __56-[UIPresentationController runTransitionForCurrentState]_block_invoke.465 ()
#14	0x00007fff4937bb72 in _runAfterCACommitDeferredBlocks ()
#15	0x00007fff4936c300 in _cleanUpAfterCAFlushAndRunDeferredBlocks ()
#16	0x00007fff4939cb09 in _afterCACommitHandler ()
#17	0x00007fff23da1087 in __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ ()
#18	0x00007fff23d9bb3e in __CFRunLoopDoObservers ()
#19	0x00007fff23d9c08a in __CFRunLoopRun ()
#20	0x00007fff23d9b8a4 in CFRunLoopRunSpecific ()
#21	0x00007fff38c05bbe in GSEventRunModal ()
#22	0x00007fff49372964 in UIApplicationMain ()
#23	0x00000001058bcc6b in main at /Volumes/Source/git/vickprat_fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/AppDelegate.swift:21
#24	0x00007fff5211c1fd in start ()
#25	0x00007fff5211c1fd in start ()

Reply:
It's not happening now.